File: Ruby_Course/Basics/methods.rb
def sayHi
  puts "Hello User"
end

puts "Top"
sayHi
puts "Bottom"

def sayHi2(name)
  puts ("Hello " + name)
end

name = "Horst"

sayHi2(name)

def sayHi3(name, age)
  puts "Hello " + name + ", you are " + age.to_s + " years old."
end

age = 55

sayHi3(name, age)

def sayHi4(name="no name", age=-1)
  puts "Hello " + name + ", you are " + age.to_s + " years old."
end

sayHi4
sayHi4(name)

File: Java_Course/src/App/Methods.java
package App;

import java.util.Scanner;

public class Methods {
	public static void methods(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		sayHi();
		sayHi2("Dave");
		sayHi2("Chris");
		
		String myName = "Dave";
		
		sayHi2(myName);
		
		sayHi3("Dave", 29);
		
		System.out.println();
	}
	
	public static void sayHi() 
	{
		System.out.println("Hello User!");
	}
	
	public static void sayHi2(String name) 
	{
		System.out.println("Hello " + name + "!");
	}
	
	public static void sayHi3(String name, int age) 
	{
		System.out.println("Hello " + name + ", you are " + age + " years old.");
	}
}
